### ISSUE TITLE
Allowable values for --ebs-volume-type should state "standard (magnetic) | io1 | gp2 | sc1 | st1"

### ISSUE THREAD

**Issue by rjhintz** [chef/knife-ec2#450]
Change [ec2_server_create.rb](https://github.com/chef/knife-ec2/blob/master/lib/chef/knife/ec2_server_create.rb) 

```rb
 option :ebs_volume_type,
        :long => "--ebs-volume-type TYPE",
        :description => "Standard or Provisioned (io1) IOPS or General Purpose (gp2)",
        :proc => Proc.new { |key| Chef::Config[:knife][:ebs_volume_type] = key },
        :default => "standard"
```

to:

```rb
 option :ebs_volume_type,
        :long => "--ebs-volume-type TYPE",
        :description => "standard (magnetic) | io1 | gp2 | sc1 | st1",
        :proc => Proc.new { |key| Chef::Config[:knife][:ebs_volume_type] = key },
        :default => "standard"
```

to align with [AWS CLI run-instances](http://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html).

This assumes the AWS EBS disk type abbreviations such as `gp2` are generally understood.

Default value should also be changed from `standard` (magnetic) to `gp2` (SSD) to align with current AWS practice. However, this is a disruptive change for current code. I'll propose this separately.

See also #451. I have to do the prerequisite admin to do a PR, but didn't want this to get lost.

**Comment by dheerajd-msys** [comment]
@rjhintz Thanks for your comments. We will get it fixed soon.

**Comment by rjhintz** [comment]
Ok, thanks. I couldn't figure out how to do a PR because the method for becoming a contributor seems obsolete. See #452

**Comment by NimishaS** [comment]
Fixed in #464
